ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/Config.hs

```diff
@@ -74,15 +75,35 @@ data SnapshotInterval =
   | RequestedSnapshotInterval DiffTime
   deriving stock (Eq, Generic, Show)
 
--- | The number of keys to read /at most/ in a backing store range query.
+-- | The number of diffs in the immutable part of the chain that we have to see
+-- before we flush ledger state to disk. See 'onDiskShouldFlush'. It can be:
```

Comment:
```suggestion
-- | The number of diffs in the immutable part of the chain that we have to see
-- before we flush the ledger state to disk. See 'onDiskShouldFlush'. It can be:
```